guides/common/assembly_installing-capsule-server.adoc

```diff
@@ -19,7 +19,7 @@ endif::[]
 
 ifdef::foreman-el,foreman-deb,satellite[]
 // Configuring Repositories
-include::modules/proc_configuring-repositories.adoc[leveloffset=+1]
+include::modules/proc_configuring-capsule-repositories.adoc[leveloffset=+1]
```

Comment:
I wonder if this is a good idea, or not.
While Satellite does have different repositories for Satellite and Capsule, upstream does not.
So maybe `proc_configuring-capsule-repositories.adoc` should be Satellite-specific?

What does @melcorr think?

Reply:
It's a very bad idea to add this. We really went to a lot of effort to remove all this.

Reply:
> I wonder if this is a good idea, or not. While Satellite does have different repositories for Satellite and Capsule, upstream does not. So maybe `proc_configuring-capsule-repositories.adoc` should be Satellite-specific?
> 
Agreed. 

@mjivraja please, never ever ever add Satellite product names to file names. This is a single source that is used in the community and in other companies. We must remain neutral.

Reply:
Resolved. Added the Capsule repos in `proc_configuring-repositories-proxy.adoc`